packages/react-components/react-nav-preview/stories/NavDrawer/NavDrawerDefault.stories.tsx

```diff
@@ -32,24 +48,58 @@ const useStyles = makeStyles({
     gridRowGap: tokens.spacingVerticalXXL,
     gridAutoRows: 'max-content',
   },
-
   field: {
     display: 'grid',
     gridRowGap: tokens.spacingVerticalS,
   },
+
+  headingContent: {
```

Comment:
Can you elaborate on these styles some? Are these styles meant to be the default visual treatment of `Nav`? Would we expect users to copy-paste them into their code?

Or put another way, why aren't these baked into the `Nav`?